recipe/meta.yaml

```diff
@@ -13,18 +13,25 @@ source:
 
 build:
   noarch: python
-  number: 0
-  # The upstream directions say to use bazel, but this is
-  # all bazel ends up doing. Since our bazel package can have
-  # glibc version issues on CI, just do things this way.
+  number: 1
   script:
-    - rm -f BUILD
-    - {{ PYTHON }} -m pip install . --no-deps -vv
-
+    - bazel build :pip_pkg
+    - mkdir tmp-pkg
+    - ./bazel-bin/pip_pkg tmp-pkg
+    - "{{ PYTHON }} -m pip install --upgrade --no-deps -vv tmp-pkg/*.whl"
+    
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - bazel 
```

Comment:
We either keep this in the build or in the host. As bazel is actually just a glorified python laucher, and we have python in the host, I would just keep bazel and its (unnecessary) dependencies in the host section. 
```suggestion
```

Reply:
Thanks